llama-index-integrations/llms/llama-index-llms-litellm/pyproject.toml

```diff
@@ -40,7 +40,7 @@ jupyter = "^1.0.0"
 mypy = "0.991"
 pre-commit = "3.2.0"
 pylint = "2.15.10"
-pytest = "7.2.1"
+pytest = "^8.3.5"
```

Comment:
had to upgrade pytest, otherwise was running into some version conflict issues between pytest-asyncio, which expects pytest to be at least version 8. This appears to be normal in this repository, with varied pytest versions per plugin